torchgen/api/types/signatures.py

```diff
@@ -286,20 +286,16 @@ def dispatcher_exprs(self) -> List[Expr]:
 
 @dataclass(frozen=True)
 class ViewInverseSignature:
-    g: NativeFunctionsViewGroup
+    f: NativeFunction
```

Comment:
nit: I think it's a tiny bit nicer to continue storing the `NativeFunctionsViewGroup` in this class, since it makes an invariant obvious: you can only ever have view aten ops inside of a `ViewInverseSignature`. If we make this `NativeFunction`, we could end up with a bug where a non-view op makes it into here.

Reply:
sure, I'll change it back. I was originally thinking I'd use this with a `NativeFunction` within `gen_in_place_or_view.py`, but I ended up using the factored-out helper `reverse_name()` directly.